README.md

```diff
@@ -56,17 +56,72 @@ See:
 * [drupal/parse-composer](https://packagist.org/packages/drupal/parse-composer)
 * [winmillwill/settings_compile](https://packagist.org/packages/winmillwill/settings_compile)
 
+## Project Customization
+
+You may want to customize a couple of things about your box first. The scripts
+are built to take most of the work out of configuration. There are pretty much
+two things you may want to do:
+
+### Change the Project Name
+
+By editing the [project variable][CPN1], you can give your project a custom
+name. Note that this will affect several other configurations in the build as
+well, including what the base module will be called and what folder the site
+will live in.
+
+[CPN1]: https://github.com/promet/drupal7-framework/blob/master/Vagrantfile#L6
+
+### Change the VM IP
+
+You can edit the [IP variable][CVMIP1] if you want to run more than one VM at a
+time.
+
+[CVMIP1]: https://github.com/promet/drupal7-framework/blob/master/Vagrantfile#L5
+
+## Getting Started Developing
+
+* You need to edit your machine's local host file. Add the entry
+  `10.33.36.11 drupalproject.dev`
+* Run `vagrant up --provision` to build the environment.
+* ssh in with `vagrant ssh`
+* Navigate to `/var/www/sites/drupalproject.dev`.
+* PARTY!!!
+
+Vagrant provision currently does a full site install.
+
+To update without rebuilding, run `build/update.sh` from the project root in
+the vagrant box.
+
+It is also worth noting, if you are working on an existing site, that the
+default install script allows you to provide a reference database in order to
+start your development. Simply add a sql file to either of the following:
+
+* `build/ref/drupalproject.sql`
+* `build/ref/drupalproject.sql.gz`
```

Comment:
Feast your eyes on this beauty!
https://github.com/generalredneck/drupal7-framework/blob/update/build/vagrant.sh#L56